src/Saas.Deployment/Saas.Deployment.Root/azuredeploy.json

```diff
@@ -151,7 +151,7 @@
       "apiVersion": "2021-04-01",
       "properties": {
         "templateLink": {
-          "uri": "https://raw.githubusercontent.com/Azure/azure-saas/main/src/Saas.Provider/Saas.Provider.Web.Deployment/azuredeploy.json"
+          "uri": "https://raw.githubusercontent.com/Azure/azure-saas/main/src/Saas.Provider/Saas.LandingSignup.Web.Deployment/azuredeploy.json"
```

Comment:
The folder name needs to be changed as well 

`/main/src/Saas.LandingSignup/Saas.LandingSignup.Web.Deployment`